src/components/CardAboutShelter/CardAboutShelter.tsx

```diff
@@ -72,7 +80,7 @@ const CardAboutShelter = (props: ICardAboutShelter) => {
           value={check(shelter.pix) ? `${shelter.pix}` : 'Não informado'}
         />
       </div>
-    </Card>
+    </Card >
```

Comment:
Sobrou um espaço aqui rs

```suggestion
    </Card>
```